readme.md

```diff
@@ -84,51 +85,59 @@ Exported definitions should be documented with [TSDoc](https://github.com/Micros
 Example:
 
 ```ts
-export interface Options {
-	/**
-	Allow negative numbers.
+/// <reference lib="dom"/>
```

Comment:
Can you add a note about when to include this, when to include the `node` reference, and when to include any third-party reference?